### PR TITLE
Fix #79: Add EvergreenTests & EvergreenUITests Schemes

### LINKED ISSUE #79: Setup EvergreenTests & EvergreenUITests Schemes
Currently there are no unit or UI tests for the Evergreen module. Setting up the EvergreenTests Scheme is the first step to improving code coverage.

### PR DESCRIPTION
**Pull request by rygood**
Added EvergreenUITests Target as well.

### CODE REVIEW

File: EvergreenUITests/EvergreenUITests.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//
```
Comment (joeblau) [inline EvergreenUITests/EvergreenUITests.swift]: Is this file necessary since it doesn't do anything yet?

File: Evergreen.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -813,19 +850,37 @@
...
 			attributes = {
-				LastSwiftUpdateCheck = 0820;
+				LastSwiftUpdateCheck = 0830;
 				LastUpgradeCheck = 0820;
 				ORGANIZATIONNAME = "Ranchero Software";
 				TargetAttributes = {
 					849C645F1ED37A5D003D8FC0 = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = M8L2WTLA8W;
+						DevelopmentTeam = PZ4HKYD5WL;
```
Comment (joeblau) [inline Evergreen.xcodeproj/project.pbxproj]: Changes like this might break @brentsimmons branch since he's likely on a different dev team than you are.

File: Evergreen.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -646,7 +664,7 @@
 				849A97961ED9EFAA007D329B /* Extensions */,
 				849A97991ED9EFB6007D329B /* Resources */,
 				84FB9A2C1EDCD6A4003D53B9 /* Frameworks */,
-				849C64741ED37A5D003D8FC0 /* EvergreenTests */,
+				9451C1F51EE0778700605914 /* Tests */,
```
Comment (joeblau) [inline Evergreen.xcodeproj/project.pbxproj]: Why did this get renamed?

Comment (rygood) [inline Evergreen.xcodeproj/project.pbxproj]: EvergreenTests and the new EvergreenUITests were moved into a new Tests group for organization.

File: EvergreenUITests/EvergreenUITests.swift (hunk 1 above)
Comment (rygood) [inline EvergreenUITests/EvergreenUITests.swift]: It's there as a first step in the process of adding UI tests to the project. Just a EvergreenTests.swift is already in the project as a starting point / placeholder for unit tests.

**Comment by rygood** [comment]
@brentsimmons are tests something that you want?
